cost/kubecost/sizing/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.2
+
+- Changed to percentile on CPU and RAM to request recommendations on, now receives a number between 1 and 100, default value set to 75 %
```

Comment:
Correct grammar so that it does not use commas to end sentences. We also don't need to get too detailed in the CHANGELOG as the specifics will be in the README. Example:

- CPU and RAM parameters now based on percentiles.

Reply:
Solved, thank you